### ISSUE TITLE
Error: Cannot find module './lib/utils'

### ISSUE THREAD

**Issue by fregante** [AJHenry/listr-better-renderer#2]
There's no `lib` in `dist` https://www.unpkg.com/browse/listr-better-renderer@0.1.0/dist/

**Comment by ajhenry** [comment]
Uh oh, that's not supposed to happen 

As for this project, I'm unfortunately not going to maintain it anymore. That's because I found [listr2](https://github.com/cenk1cenk2/listr2) to be a much, _much_ better written listr implementation with a more maintainable renderer. I have already contributed all of the features found in this project into the default listr2 renderer and advise you to switch to that project moving forward

Cheers!
